src/main/java/QueryServlet.java

```diff
@@ -97,9 +121,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       return;
     }
 
-    String dataRow = queryToDataRow.get(action).get(personType);
+    String dataRow;
```

Comment:
How about 

String dataRow = queryToDataRowGeneric.get(action).get(personType);
if (year > 2013
        && queryToDataRowPost2013.containsKey(action)
        && queryToDataRowPost2013.get(action).containsKey(personType)) {
      dataRow = queryToDataRowPost2013.get(action).get(personType);
}

Then you don't need the else and its easier to read.

Reply:
Good idea, thank you! Done.